cmd/sales-api/main.go

```diff
@@ -78,10 +75,9 @@ func run(log *log.Logger) error {
 			Algorithm      string `conf:"default:RS256"`
 		}
 		Zipkin struct {
-			LocalEndpoint string  `conf:"default:0.0.0.0:3000"`
-			ReporterURI   string  `conf:"default:http://zipkin:9411/api/v2/spans"`
-			ServiceName   string  `conf:"default:sales-api"`
-			Probability   float64 `conf:"default:0.05"`
```

Comment:
Zipkin struct remove Probability property ?

Reply:
I'm passing that into Init because of your comment about not really wanting to sample all.

Reply:
Init function not use this property